researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/BridgeDataProvider.java

```diff
@@ -561,6 +561,24 @@ public void setUserSharingScope(Context context, String scope) {
                 .toObservable();
     }
 
+    /**
+     * Make participant data available for download.
+     * <p>
+     * Request the uploaded data for this user, in a given time range (inclusive). Bridge will
+     * asynchronously gather the user's data for the given time range and email a secure link to the
+     * participant's registered email address.
+     *
+     * @param startDate The first day to include in reports that are returned (required)
+     * @param endDate   The last day to include in reports that are returned (required)
+     * @return completable
+     */
+    @NonNull
+    public Observable<Message> downloadData(LocalDate startDate,
```

Comment:
The data provider generally returns only DataResponse objects

Reply:
Oh right, my fault.  I'll change that.